lib/SVF-FE/LLVMModule.cpp

```diff
@@ -200,6 +201,14 @@ void LLVMModuleSet::build()
                         svffun->getPostDomTreeMap()[bb] = Set<const BasicBlock *>();
                     }
                 }
+                const Loop *loop = loopInfo.getLoopFor(bb);
```

Comment:
move this line to the following if branch condition
if(const Loop *loop = loopInfo.getLoopFor(bb)){

}